cirq/google/api/v2/program.proto

```diff
@@ -24,6 +24,14 @@ message Program {
   }
 }
 
+// A Batch of multiple circuits that should be run as one program
```

Comment:
Might want to clarify here that "as one program" is at the level of the quantum engine api. These will not _necessarily_ be executed all at once on the hardware.

Reply:
Done.